src/test/regress/sql/multi_extension.sql

```diff
@@ -193,6 +193,15 @@ SELECT * FROM print_extension_changes();
 ALTER EXTENSION citus UPDATE TO '9.4-1';
 SELECT * FROM print_extension_changes();
 
+ALTER EXTENSION citus UPDATE TO '9.5-1';
```

Comment:
Please add a small comment here for consistency
```suggestion
-- Test downgrade to 9.4-1 from 9.5-1
ALTER EXTENSION citus UPDATE TO '9.5-1';
```